modules/tinymce/src/core/test/ts/browser/FormatterApplyTest.ts

```diff
@@ -1771,27 +1771,16 @@ describe('browser.tinymce.core.FormatterApplyTest', () => {
     assert.equal(editor.getContent(), '<p>abc</p><p contenteditable="false"><span contenteditable="true"><b>def</b></span></p>', 'Text is bold');
   });
 
-  it('Del element wrapping blocks', () => {
+  it('Div element wrapping blocks', () => {
     const editor = hook.editor();
     editor.setContent('<p>a</p>');
     LegacyUnit.setSelection(editor, 'p', 0, 'p', 1);
     editor.formatter.register('format', {
-      block: 'del',
+      block: 'div',
       wrapper: true
     });
     editor.formatter.apply('format');
-    assert.equal(getContent(editor), '<p><del><p>a</p></del></p>');
```

Comment:
Del is one of those weird elements where it can be a inline or a block depending on context. So in this particular case it should be a inline and not allowed to contain that P. At some point we should start looking into supporting these things it's similar to block/inline `a` elements.